lib/fast_counter.ex

```diff
@@ -17,7 +17,7 @@ defmodule Instruments.FastCounter do
 
   use GenServer
 
-  def start_link() do
+  def start_link(_ \\ :ok) do
```

Comment:
It's a little bit odd to add a default to an unused var.  Recommend removing as it has no effect.